packages/blade/src/components/Tabs/__tests__/Tabs.web.test.tsx

```diff
@@ -0,0 +1,442 @@
+import userEvent from '@testing-library/user-event';
+import { fireEvent, screen } from '@testing-library/react';
+import React from 'react';
+import { Tabs, TabList, TabItem, TabPanel } from '../';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+import { Text } from '~components/Typography';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import assertAccessible from '~utils/testing/assertAccessible.web';
+
+const queryTabPanelByText = (text: string): HTMLElement | null => {
+  return (
+    screen.queryAllByRole('tabpanel', { hidden: true }).find((el) => el.textContent === text) ??
+    null
+  );
+};
+
+describe('Tabs', () => {
+  it('should render', () => {
+    const { baseElement, queryAllByRole } = renderWithTheme(
+      <Tabs defaultValue="refunds">
+        <TabList>
+          <TabItem value="payments">Payments</TabItem>
+          <TabItem value="refunds">Refunds</TabItem>
+        </TabList>
+
+        <TabPanel value="payments">
+          <Text>Payments</Text>
+        </TabPanel>
+        <TabPanel value="refunds">
+          <Text>Refunds</Text>
+        </TabPanel>
+      </Tabs>,
+    );
+
+    expect(baseElement).toMatchSnapshot();
+
+    // `queryByRole` is failing but `queryAllByRole` is working
```

Comment:
this needs to be communicated to consumers as well since they'll also be writing tests

Reply:
Where/How do we communicate this?